### DISCUSSION TITLE
GADM: 全球行政区划数据库

### DISCUSSION THREAD

**Discussion by JandyZ22** [gmt-china/GMT_docs#991] [Q&A]
In this section of the manual
Command:
ogr2ogr -f OGR_GMT '' gadm36_USA.gpkg gadm36_USA_0
ogr2ogr -f OGR_GMT '' gadm36_USA.gpkg gadm36_USA_1
ogr2ogr -f OGR_GMT '' gadm36_USA.gpkg gadm36_USA_2
->Unable to find driver `OGR_GMT'.

Should this be changed to:
ogr2ogr -f GMT '' gadm36_USA.gpkg gadm36_USA_0
ogr2ogr -f GMT '' gadm36_USA.gpkg gadm36_USA_1
ogr2ogr -f GMT '' gadm36_USA.gpkg gadm36_USA_2

**Comment by ZMAlt** [comment]
这种问题可以放到  issue 里。

确定吗？ ogr2ogr usage 里仍然是 OGR_GMT
```
ogr2ogr --long-usage | grep GMT          
     -f "OGR_GMT"
```

**Reply by JandyZ22** [reply]
我服务器gmt版本是6.3(64-bit)，系统是centos 7

> (base) [ydz@gaia Documents]$ ogr2ogr --long-usage | grep GMT 
     -f "GMT"

**Reply by ZMAlt** [reply]
这个和 gmt 版本没什么关系，ogr2ogr 命令来自 GDAL，要看 GDAL。我的 GDAL 是 3.5.2，是最新版。

**Reply by JandyZ22** [reply]
> gdalinfo --version
> GDAL 1.11.4, released 2016/01/25
原来如此，centos 7 yum 安装的老版的GDAL ，谢谢你的解答！
